requirements.txt

```
@@ -1,8 +1,2 @@
 setuptools
-geopy>=1.14.0
-docopt
-pyyaml
-certifi
-amqtt>=0.10.0,<0.11.0
-websockets>=9.0
 aiohttp
```

```suggestion
aiohttp<4.0
```
`aiohttp` is a bigger package. If they do a major version change, it's likely to cause some issues or at least some validation should happen before the dependency is bumped. (See also #81, although normally pip doesn't install alpha packages.)